src/coredump/coredump.c

```diff
@@ -1560,8 +1613,64 @@ static int forward_coredump_to_container(Context *context) {
         return 0;
 }
 
+static int gather_pid_mount_tree_fd(const Context *context) {
+        _cleanup_close_ int mntns_fd = -EBADF, root_fd = -EBADF;
+        _cleanup_close_pair_ int pair[2] = EBADF_PAIR;
+        int fd = -EBADF, r;
+        pid_t child;
+
+        assert(context);
+
+        if (!arg_access_container)
+                return -EBADF;
+
```

Comment:
given all this work is pointless if the new library is not available, I'd add something like:

```
#if !HAVE_DWFL_SET_SYSROOT
return -EBADF;
#endif
```